base/sysinfo.jl

```diff
@@ -141,12 +141,22 @@ function cpu_info()
     return cpus
 end
 
+"""
+    uptime()
+
+Gets the current system uptime.
```

Comment:
These are exported from `Sys` but `Base` does not do `using Sys` or `importall Sys` so they always need the prefix.